.github/workflows/distcheck.yml

```diff
@@ -0,0 +1,35 @@
+name: Autotools distcheck
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+
+jobs:
+  distcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Linux dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libtool-bin libogg-dev doxygen libxml2-utils w3c-sgml-lib
+
+      - name: Build with Autotools
+        run: |
+          ./autogen.sh
+          ./configure
+          make
```

Comment:
```suggestion
```
I think this can be removed, now everything is build twice. While it doesn't take long, it makes the logs unnecessarily long